x-pack/filebeat/input/httpjson/request.go

```diff
@@ -660,21 +663,21 @@ func (r *requester) processChainPaginationEvents(ctx context.Context, trCtx *tra
 			// reformat urls of requestFactory using ids
 			rf.url, err = generateNewUrl(rf.replace, urlString, id)
 			if err != nil {
-				return -1, fmt.Errorf("failed to generate new URL: %w", err)
+				return -1, fmt.Errorf("failed to generate new url: %w", err)
```

Comment:
The error message seems to be same as https://github.com/elastic/beats/pull/36529/files#diff-ffe24395246b343e8006e4eaef52c2d3bf68c20497ac96ef1e626d4351258047R155. Since this is specific for chain requests, should we change it to `failed to generate new url for chain pagination request`? Ofcourse, we also have the line number inside the error trace. Just checking..